docs/en/solutions/apply-catboost-model.md

```diff
@@ -0,0 +1,212 @@
+
+```bash
+$ clickhouse client
+```
+
+> **Note:** The ClickHouse server is already running inside the Docker container.
+
+**2.** Create the table using the command:
+
+```sql
+:) CREATE TABLE amazon_train
+(
+    date Date MATERIALIZED today(), 
+    ACTION UInt8, 
+    RESOURCE UInt32, 
+    MGR_ID UInt32, 
+    ROLE_ROLLUP_1 UInt32, 
+    ROLE_ROLLUP_2 UInt32, 
+    ROLE_DEPTNAME UInt32, 
+    ROLE_TITLE UInt32, 
+    ROLE_FAMILY_DESC UInt32, 
+    ROLE_FAMILY UInt32, 
+    ROLE_CODE UInt32
+)
+ENGINE = MergeTree(date, date, 8192)
+```
+
+## 2. Insert the data to the table {#insert-the-data-to-the-table}
+
+To insert the data:
+
+**1.** Exit from ClickHouse:
+
+```sql
+:) exit
+```
+
+**2.** Upload the data:
+
+```bash
+$ clickhouse client --host 127.0.0.1 --query 'INSERT INTO amazon_train FORMAT CSVWithNames' < ~/amazon/train.csv
+```
+
+**3.** Make sure the data has been uploaded:
+
+```sql
+$ clickhouse client
+:) SELECT count() FROM amazon_train
+SELECT count()
+FROM amazon_train
+
++-count()-+
+|   32769 |
++---------+
+```
+
+## 3. Configure the model to work with the trained model {#configure-the-model}
+
+This step is optional: the Docker container contains all configuration files. 
+
+**1.** Create a config file (for example, `config_model.xml`) with the model configuration:
+
+```xml
+<models>
+    <model>
+        <!-- Model type. Now catboost only. -->
+        <type>catboost</type>
+        <!-- Model name. -->
+        <name>amazon</name>
+        <!-- Path to trained model. -->
+        <path>/home/catboost/tutorial/catboost_model.bin</path>
+        <!-- Update interval. -->
+        <lifetime>0</lifetime>
+    </model>
+</models>
+```
+
+> **Note:** To show contents of the config file in the Docker container, run `cat models/amazon_model.xml`.
+
+**2.** Add the following lines to the `/etc/clickhouse-server/config.xml` file:
+
+```xml
+<catboost_dynamic_library_path>/home/catboost/.data/libcatboostmodel.so</catboost_dynamic_library_path>
+<models_config>/home/catboost/models/*_model.xml</models_config>
+```
+
+> **Note:** To show contents of the ClickHouse config file in the Docker container, run `cat ../../etc/clickhouse-server/config.xml`.
+
+**3.** Restart ClickHouse server:
+
+```bash
+$ sudo service clickhouse-server restart
+```
+
+## 4. Test the trained model {#test-the-trained-model}
+
+For test run the ClickHouse client `$ clickhouse client`.
+
+- Let's make sure that the model is working:
+
+```sql
+:) SELECT 
+    modelEvaluate('amazon', 
+                RESOURCE,
+                MGR_ID,
+                ROLE_ROLLUP_1,
+                ROLE_ROLLUP_2,
+                ROLE_DEPTNAME,
+                ROLE_TITLE,
+                ROLE_FAMILY_DESC,
+                ROLE_FAMILY,
+                ROLE_CODE) > 0 AS prediction, 
+    ACTION AS target
+FROM amazon_train
+LIMIT 10
+```
+
+> **Note:** Function `modelEvaluate` returns tuple with per-class raw predictions for multiclass models.
```

Comment:
Example of what it returns? Link to its standalone documentation?

Reply:
`modelEvaluate` description only available in English version.

Reply:
> modelEvaluate description only available in English version.

let's fix that as well?

and queries still need some examples of what they return